src/components/utils/isDesktop.js

```diff
@@ -1,3 +1,3 @@
-const width = (window.matchMedia("(min-width: 800px)").matches)
```

Comment:
I do have that indirection as well at:

```js
supportsWebcam (){...
```

but since it's only used in this module I did not move it to some place separate.